scripts/ci/batch_pw_load.sh

```
@@ -0,0 +1,149 @@
+fi
+COMPOSE_ARGS="$COMPOSE_ARGS -f $DOCKER_COMPOSE_TEST_FILE"
+
+# Ensure required compose env vars have sane defaults for healthchecks
+REACT_APP_MOCKOON_PORT=${REACT_APP_MOCKOON_PORT:-"8080"}
+GRAPHQL_PORT=${GRAPHQL_PORT:-"4000"}
+GRAPHQL_API_PATH=${GRAPHQL_API_PATH:-"graphql"}
+export REACT_APP_MOCKOON_PORT GRAPHQL_PORT GRAPHQL_API_PATH REACT_APP_PROD_PORT
+
+:
+
+PLAYWRIGHT_ENV_FLAGS="\
```

Shellcheck (0.11.0)

[warning] PLAYWRIGHT_ENV_FLAGS appears unused. Verify use (or export if used externally).

(SC2034)